# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/svg/crashtests/crashtests.list
# Commit: 9f710d42c811
# Full Hash: 9f710d42c811206544a54f51a97897d7d597ed0b
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-12-06 16:58:51
# Description:
#   Bug 1836831 - don't try to resolve ray with StrokeBox when non-scaling-stroke is in effect r=boris
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195552
# ==============================================================================

diff -r 1be0a23195c0 -r 9f710d42c811 layout/svg/crashtests/crashtests.list
--- a/layout/svg/crashtests/crashtests.list	Wed Dec 06 09:14:24 2023 +0000
+++ b/layout/svg/crashtests/crashtests.list	Wed Dec 06 09:19:01 2023 +0000
@@ -262,5 +262,6 @@
 load 1810260.html
 load 1826444-1.html
 load 1831419.html
+load 1836831.html
 load 1840195-1.html
 load 1848851.html
